Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnStatus.hs

```diff
@@ -0,0 +1,36 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module Beckn.ACL.OnStatus (mkOnStatusMessage) where
+
+import qualified Beckn.Types.Core.Taxi.OnStatus as OnStatus
+import qualified Domain.Action.Beckn.Status as DStatus
+import qualified Domain.Types.Ride as DRide
+import Kernel.Types.Id (Id)
+
+mkOnStatusMessage :: Id DRide.Ride -> DStatus.DStatusRes -> OnStatus.OnStatusMessage
+mkOnStatusMessage rideId res = do
+  OnStatus.OnStatusMessage
+    { order =
+        OnStatus.Order
+          { id = rideId.getId,
```

Comment:
order.id is bookingId and fulfilmentId is rideId

Reply:
@olegromashin Can you do these changes ?

Reply:
Done